app/configs/fuzz_features.conf

```diff
@@ -0,0 +1,48 @@
+# The goal of this file is to:
+# 1. Fuzz more code
+# 2. Reduce the gap between fuzzed SOF and the real thing.
+
+# KConfig warnings are NOT fatal so you must always INSPECT build logs when changing
+# .conf files. See https://github.com/thesofproject/sof/issues/9386
+
+# Note 1. is not as simple as enabling as many CONFIG_ as possible. Enabling some CONFIG_
+# can technically _disable_ some code paths. But the opposite is more common so let's add
+# more.
+
+# In the longer term we should have some more elaborate configuration framework to reduce
+# duplicate/diverge between fuzzing and production but also across product
+# generations. Something like config fragments including each other? Not something as
+# complicated as Yocto fragments but something more flexible than
+# https://docs.zephyrproject.org/latest/build/kconfig/setting.html#initial-conf
+#
+# Discuss in https://github.com/thesofproject/sof/issues/9386
+
+# Many of these features are too far from IPC to make any fuzzing difference. But as long
+# as they reduce the size of the textual difference between build-fuzz/zephyr/.config and
+# build-xxx/zephyr/.config, they make that manual comparison easier which is still a win.
+
+
+CONFIG_COUNTER=y
+
+CONFIG_PROBE=y
+
+CONFIG_CRYPTO=y
+
+CONFIG_LOG_TIMESTAMP_64BIT=y
+CONFIG_MM_DRV=y
+
+CONFIG_DMA=y
+
+CONFIG_DAI=y
+
+CONFIG_PM_DEVICE=y
+CONFIG_POWER_DOMAIN=y
+
+CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
+CONFIG_COMP_CROSSOVER=y
+CONFIG_COMP_DRC=y
+CONFIG_COMP_KPB=y
```

Comment:
everything should be one here, even if its only a stub, can you turn everything on?

Reply:
I ran into some Kconfig warnings when trying to enable more stuff and I'm running a bit out of SOF time... the main purpose of this PR is really to set the scripting "framework" in place so people who know better can come and just tweak the CONFIG_... / features does that make sense?

Reply:
yep that is fine

Reply:
A long time ago, I tried to find some -Werr option in kconfiglib.py but there is none, at least none for this. Because most people don't look at build logs, I'm afraid it means that CONFIG_ warnings and messes will never stop creeping in, see some examples in:
- #9386